src/main/java/core/selenium/TrabajopolisConfig.java

```diff
@@ -68,9 +69,12 @@ public Properties config() {
             urlLogin = properties.getProperty("urlLogin");
             urlProfile = properties.getProperty("urlProfile");
             urlBase = properties.getProperty("urlBase");
+            urlCurriculum = properties.getProperty("urlCurriculum");
+            urlFillCurriculum = properties.getProperty("urlFillCurriculum");
         } catch (FileNotFoundException e) {
             e.printStackTrace();
             throw new NullPointerException("This file not exist!!");
```

Comment:
Please set the type of throw to something more acord with the type you are catching, maybe it could be "RunTimeException".

Reply:
done